tests/functional/api/test_groups.py

```diff
@@ -35,6 +35,15 @@ def test_it_returns_http_400_with_invalid_payload(self, app, token_auth_header):
 
         assert res.status_code == 400
 
+    def test_it_returns_http_400_if_groupid_set_on_default_authority(self, app, token_auth_header):
```

Comment:
This test is wrong, I just noticed. It's 400-ing off of that `groupid` format, not authority, so it's not testing what it says it's testing. Noted.